client/src/components/Panels/Common/ToolAdvancedSearch.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+    <ToolsProvider
+        v-slot="{ loading, result: itemsLoaded, count: totalItems }"
+        :filter-settings="filterSettings"
+        :show-help="showHelp === 'true'">
+        <section>
+            <div class="mb-2">
+                <span class="row">
+                    <span class="col">
+                        <h4 class="d-inline-block align-middle">Advanced Tool Search Results</h4>
+                        <b-button v-if="hasFilters" class="float-right" size="sm" @click="onClear">
```

Comment:
Why do we need this button?

Reply:
Initially, I was planning to clear up the filter fields by pressing on this but now I realize that might be pointless, going to get rid of it